examples/hypercompute_clusters/a3u-gke-gcs/a3u-gke-gcs.yaml

```diff
@@ -0,0 +1,296 @@
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+blueprint_name: a3u-gke-gcs
+
+vars:
+  project_id: # Insert GCP project
+  deployment_name: # Unique name of this cluster, like a3u-gke-gcs
+  region: # Region, e.g. europe-west1
+  zone: # Zone, e.g. europe-west1-b
+
+  # Cidr block containing the IP of the machine calling terraform and kubectl
+  # The value can be more specific if the IPs are known which will run kubectl
+  # e.g. the local system running Terraform or a remote node
+  authorized_cidr: 0.0.0.0/0
```

Comment:
Should this be kept as the default value? instead of the traditional 
`<your-ip-address>/32` in other blueprints?

Reply:
I feel keeping it as <your-ip-address>/32 can give user a better consistent experience.
they can always set it to 0.0.0.0/0 to open it up (maybe have a comment in the blueprint for this?)